atmo/users/migrations/0002_rewrite_usernames.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-04-26 09:39
+from __future__ import unicode_literals
+
+from django.db import migrations
+from allauth.utils import generate_unique_username
+from allauth.account.utils import user_username, user_email, user_field
+
+
+def fix_usernames(apps, schema_editor):
+    User = apps.get_model('auth', 'User')
+    all_usernames = User.objects.all().values_list('username', flat=True)
```

Comment:
Good catch, that's a leftover from when I wasn't using `generate_unique_username` which does this behind the scenes.